tests/api2/test_audit_basic.py

```diff
@@ -115,29 +116,19 @@ def init_audit():
 
 
 @pytest.fixture(scope='class')
-def standby_user():
-    """ HA system: Create a user on the BACKUP node
-    This will generate a 'create' audit entry, yield,
-    and on exit generate a 'delete' audit entry.
+def standby_audit_event():
+    """ HA system: Create an audit event on the standby node
+    Attempt to delete a built-in user on the standby node
     """
-    user_id = None
+    event = "user.delete"
+    username = "backup"
+    user = call('user.query', [["username", "=", username]], {"select": ["id"], "get": True})
     try:
-        name = "StandbyUser" + PASSWD
-        user_id = call(
-            'failover.call_remote', 'user.create', [{
-                "username": name,
-                "full_name": name + " Deleteme",
-                "group": 100,
-                "smb": False,
-                "home_create": False,
-                "password": "testing"
-            }],
-            {'raise_connect_error': False, 'timeout': 2, 'connect_timeout': 2}
-        )
-        yield name
-    finally:
-        if user_id is not None:
-            call('failover.call_remote', 'user.delete', [user_id])
+        # This _should_ fail.  Assert if not!
+        call('failover.call_remote', event, [user['id']])
+        assert True, "!!! Should not be allowed to delete a built-in user !!!"
```

Comment:
`assert True` will do nothing, what is this supposed to do?

Reply:
The goal is to generate an audit entry on the remote node.  To generate the entry I'm attempting to delete a built-in user. 
OK, let me refactor this.